changelog.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 2.1.2
+- fix of usage `??` with `File->data` and `Group->data` see [error description](https://bugs.php.net/bug.php?id=71359) Now its safe to use `$data = File->data ?? $defaultData;` or `$data = Group->data ?? $defaultData;`
```

Comment:
- add `__isset` to classes that have `__get` that fixes class behaviours in PHP 7.0.6+